spec/javascripts/modules/remove-filter-spec.js

```diff
@@ -37,14 +37,14 @@ describe('remove-filter', function () {
 
   var $dropdown = $(
     '<div data-module="remove-filter">' +
-      '<button href="/search/news-and-communications?[][]=level_one_taxon&amp;[][]=ba3a9702-da22-487f-86c1-8334a730e559&amp;[][]=level_two_taxon&amp;[][]" class="remove-filter" role="button" aria-label="Remove filter Entering and staying in the UK" data-module="remove-filter-link" data-facet="level_one_taxon" data-value="ba3a9702-da22-487f-86c1-8334a730e559" data-track-label="Entering and staying in the UK" data-name="">✕</button>' +
+      '<button href="/search/news-and-communications?[][]=dropdown&amp;[][]=ba3a9702-da22-487f-86c1-8334a730e559&amp;[][]=level_two_taxon&amp;[][]" class="remove-filter" role="button" aria-label="Remove filter Entering and staying in the UK" data-module="remove-filter-link" data-facet="dropdown" data-value="ba3a9702-da22-487f-86c1-8334a730e559" data-track-label="Entering and staying in the UK" data-name="">✕</button>' +
```

Comment:
What's the reason behind this change? It seems like it's changing the name of something, which is reflected further on in the file, but I can't see why.

Reply:
it's basically because all three elements need to exist independently to make the tests pass. Because of the move to JS we now have to really care if the elements exist on the page before removing them otherwise it falls over. This is so that all three elements (`dropdown`, `level_one_taxon` and `level_two_taxon`) exist separately and can be used by the tests

Reply:
Fixed after a chat :)